### PR TITLE
Fix broken links in guidance

### PR DESCRIPTION
**Pull request by taylorsj1980**

### CODE REVIEW

File: module/Application/view/application/how-primary-attorneys-make-decision/index.twig [hunk 1]
```diff
@@ -88,7 +88,7 @@
     <div class="accordion-info">
 		<div class="text">
 			<p>Your choice here is very important as it affects how your LPA can be used. Whichever option you choose, your attorneys must always act in your best interests.</p>
-			<p><a class="js-guidance" href="/guide#topic-how-attorneys-make-decisions" data-journey-click="stageprompt.lpa:help:how-attorneys-make-decisions">Get help with making this decision</a></p>
+			<p><a class="js-guidance" href="/guide#topic-how-multiple-attorneys-make-decisions" data-journey-click="stageprompt.lpa:help:how-attorneys-make-decisions">Get help with making this decision</a></p>
```
Comment (elvisciotti) [inline module/Application/view/application/how-primary-attorneys-make-decision/index.twig]: @taylorsj1980 data-journey-click <-- does that need to be updated too ?

File: module/Application/view/application/how-primary-attorneys-make-decision/index.twig [hunk 2]
```diff
@@ -109,8 +109,8 @@
                 {{ formElement(how) }}
                 <div id="decisions-details" class="panel panel-border-narrow js-hidden">
 			              <p class="text">
-                        Please tell us which decisions have to be made jointly, and which can be made jointly and severally. 
-                        <a class="js-guidance" href="/guide#topic-how-attorneys-make-decisions" data-journey-click="stageprompt.lpa:help:how-attorneys-make-decisions">
+                        Please tell us which decisions have to be made jointly, and which can be made jointly and severally.
+                        <a class="js-guidance" href="/guide#topic-how-multiple-attorneys-make-decisions" data-journey-click="stageprompt.lpa:help:how-attorneys-make-decisions">
```
Comment (elvisciotti) [inline module/Application/view/application/how-primary-attorneys-make-decision/index.twig]: same as above

File: module/Application/view/application/how-primary-attorneys-make-decision/index.twig (hunk 1 above)
Comment (taylorsj1980) [inline module/Application/view/application/how-primary-attorneys-make-decision/index.twig]: I have fixed and pushed

File: module/Application/view/application/how-primary-attorneys-make-decision/index.twig (hunk 2 above)
Comment (taylorsj1980) [inline module/Application/view/application/how-primary-attorneys-make-decision/index.twig]: I have fixed and pushed

**Pull request merged**